### PR TITLE
feat: general settings drawer 

### PR DESCRIPTION
**Pull request by aoltean-plenty**

### CODE REVIEW

File: apps/web/components/SettingsView/SettingsView.vue [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    <header class="flex items-center justify-between px-4 py-5 border-b">
+      <div class="flex items-center text-xl font-bold">General Settings</div>
+      <button data-testid="design-view-close" class="!p-0" @click="closeDrawer">
+        <SfIconClose />
+      </button>
+    </header>
+
+    <UiAccordionItem
+      v-model="branding"
+      data-testid="color-section"
```
Comment (ksted) [inline apps/web/components/SettingsView/SettingsView.vue]: **todo**: Update test ID name. Please also double-check the other test IDs in the file, there's a number of duplicates.

File: apps/web/components/SettingsView/SettingsView.vue [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+      summary-class="w-full hover:bg-neutral-100 px-4 py-5 flex justify-between items-center select-none border-b"
+    >
+      <template #summary>
+        <h2 class="">Branding</h2>
+      </template>
+      <div class="py-2">
+        <div class="flex justify-between mb-2">
+          <UiFormLabel>Logo</UiFormLabel>
+          <SfTooltip
+            label="The logo is displayed in the header of the onlineshop. For the best performance, you should choose an image file in one of the following formats: SVG, AVIF or WebP. If you choose SVG, the size must be 150 x 40 px. For other formats, the maximum size is 180 px (width) by 80 px (height)."
```
Comment (ksted) [inline apps/web/components/SettingsView/SettingsView.vue]: **suggestion**: Move the SVG sizes down to the `span`. I think it's confusing to get size information in two different places?

File: apps/web/components/SettingsView/SettingsView.vue [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+            >Ideal dimensions: 150 × 40 px, or a maximum width of 180 px and a maximum height of 80 px.</span
+          >
+        </label>
+      </div>
+
+      <div class="py-2">
+        <div class="flex justify-between mb-2">
+          <UiFormLabel>Favicon</UiFormLabel>
+          <SfTooltip
+            label="A favicon helps customers recognize your site in browser tabs and bookmarks. It must be an ICO file, as other image formats are not supported."
```
Comment (ksted) [inline apps/web/components/SettingsView/SettingsView.vue]:
```suggestion
            label="A favicon helps customers recognize your site in browser tabs and bookmarks. Required file format: .ico"
```

File: apps/web/components/SettingsView/SettingsView.vue [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+            class="ml-2 z-10"
+          >
+            <SfIconInfo :size="'sm'" />
+          </SfTooltip>
+        </div>
+        <label>
+          <SfInput v-model="favicon" type="text" placeholder="Enter URL of the image" data-testid="logo-field" />
+
+          <span class="typography-text-xs text-neutral-700"
+            >Recommended dimensions: A square of 32 × 32 px or 48 × 48 px (180 × 180 px for iPhones/iPads)
```
Comment (ksted) [inline apps/web/components/SettingsView/SettingsView.vue]: **question**: Does it make sense to differentiate based on device here? We only have one input field.

File: apps/web/nuxt.config.ts [hunk 5]
```diff
@@ -68,6 +68,10 @@ export default defineNuxtConfig({
       showConfigurationDrawer: process.env.SHOW_CONFIGURATION_DRAWER === '1',
       defaultItemsPerPage: Number(process.env.DEFAULT_FEEDBACK_ITEMS_PER_PAGE ?? 10),
       headerLogo: process.env.LOGO || '/_nuxt-plenty/images/logo.svg',
+      favicon: process.env.FAVICON || '/_nuxt-plenty/favicon.ico',
+      ogTitle: process.env.OG_TITLE || 'PlentyONE Shop',
+      ogImg: process.env.OG_IMG || '/_nuxt-plenty/images/logo.svg',
+      manufactured: process.env.MANUFACTURED || '',
```
Comment (ksted) [inline apps/web/nuxt.config.ts]: These may not have any effect here. I think Favicon, OG Title and OG Image need to go into `app.config.ts` like in #1245

File: apps/web/components/SettingsView/SettingsView.vue (hunk 4 above)
Comment (aoltean-plenty) [reply apps/web/components/SettingsView/SettingsView.vue]: I think this is a question for Sabine.

File: apps/web/components/SettingsView/SettingsView.vue (hunk 2 above)
Comment (aoltean-plenty) [reply apps/web/components/SettingsView/SettingsView.vue]: This is a topic we should get Sabine involved in. In the card I have tooltip text and I have text to be added in the span bellow the input. I followed the acceptance criteria here.

File: apps/web/composables/useSiteConfiguration/useSiteConfiguration.ts [hunk 6]
```diff
@@ -34,6 +34,12 @@ export const useSiteConfiguration: UseSiteConfigurationReturn = () => {
     currentFont: useRuntimeConfig().public.font,
     primaryColor: useRuntimeConfig().public.primaryColor,
     secondaryColor: useRuntimeConfig().public.secondaryColor,
+    headerLogo: useRuntimeConfig().public.headerLogo,
+    favicon: String(useRuntimeConfig().public.favicon),
+    ogTitle: String(useRuntimeConfig().public.ogTitle),
+    ogImg: String(useRuntimeConfig().public.ogImg),
```
Comment (ksted) [inline apps/web/composables/useSiteConfiguration/useSiteConfiguration.ts]: **issue**: These are no longer part of the runtime configuration, are they? I would suggest exporting them from `app.config`, similarly to how the meta data gets exported.

**suggestion (non-blocking)**: I would also suggest clustering `ogTitle` and `ogImg` into an `openGraph` object since they belong together.

**Review by ksted — APPROVED** [review]
